### ISSUE TITLE
Vulvar abscess

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6653]
**Preferred term label:**
Vulvar abscess

**Synonyms**
Vulva abscess

**Definition (free text, please give PubMed ID)**
Abscesses involving the vulvar region. 
PMID: 27617247

**Parent term (use hpo.jax.org/app)**
Abscess

**Comment by pnrobinson** [comment]
added term
